cedar-policy-cli/src/lib.rs

```diff
@@ -649,6 +658,25 @@ pub fn link(args: &LinkArgs) -> CedarExitCode {
     }
 }
 
+pub fn visualize(args: &VisualizeArgs) -> CedarExitCode {
+    match load_entities(&args.entities_file, None) {
+        Ok(entities) => match entities.to_dot_str() {
+            Ok(dot_str) => {
+                println!("{}", dot_str);
+                CedarExitCode::Success
+            }
+            Err(e) => {
+                eprintln!("{}", e);
```

Comment:
Let's follow the pattern set elsewhere in this file and print as a `miette::Report` in Debug format.
```suggestion
                let report = miette::Report::new(e);
                eprintln!("{report:?}");
```

Reply:
On closer reading of the code, this error should actually never occur. The `write_str` in the dot code should always succeed I believe.
What do we do with errors that should not occur? I can make a new `EntitiesError` variant, or re-write the code to not use `write_str` and instead use a vec of strings and a join at the end.

Reply:
I think it's fine to have an error that "should never occur" in this case, since it's easy to report it correctly in the hypothetical case that it did occur.  But rewriting to avoid `write_str` would be fine too.

Reply:
Any reason not to switch it to using `push_str` so we can get rid of the `Result`? I'd like that better if we're adding it to our public API.

Reply:
Good idea! Done